.gitignore

```diff
@@ -1,3 +1,4 @@
+testSerial.py
```

Comment:
I think Adit may just be unclear about what the git ignore file is for.  I'll clarify for him.  He probably wrote this as a test file and didn't want to commit it.